app/policies/comment_policy.rb

```diff
@@ -1,15 +1,24 @@
 class CommentPolicy < ApplicationPolicy
-  DELETE_COMMENT_ROLES = %w(superadmin policy_and_abuse communications support).freeze
+  DESTROY_ROLES = %w(superadmin policy_and_abuse communications support).freeze
+  SPAM_ROLES = %w(superadmin policy_and_abuse communications support).freeze
 
-  def self.can_delete_comment?(user)
-    self.new(user, nil).can_delete_comment?
+  def self.can_destroy_comment?(user)
```

Comment:
The 2 static methods here are unused. I think we only need the static methods in the case of UserCreationPolicy (which handles multiple models and cannot be looked up by the `policy` view helper).

Everything else looks good.